niceman/resource/shell.py

```diff
@@ -86,6 +86,13 @@ def mkdir(self, path, parents=False):
             else:
                 os.mkdir(path)
 
+    def get(self, src_path, dest_path, owner=None, group=None):
+        pass
```

Comment:
I think, ideally, in case if owner and/or group specified and do not match (thus do a check) we should issue a warning that nothing was changed but they differ (here and in put)